### PR TITLE
Optimize

### PR DESCRIPTION
**Pull request by alextio**
Made tweaks and adjustments based on various feedback.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,9 @@
     "react": "^18",
     "react-dom": "^18",
     "react-icons": "^5.0.1",
-    "react-markdown": "^9.0.1"
+    "react-intersection-observer": "^9.13.1",
+    "react-markdown": "^9.0.1",
```
Comment (jhw123) [inline package.json:27]: Do we still use these two packages?

File: public/members/default.png
Comment (jhw123) [inline public/members/default.png:1]: didn't we have a cute image that Yoomin made previously? I remember that I saw it on the Figma

File: yarn.lock [hunk 2]
```diff
@@ -2949,7 +2949,9 @@ __metadata:
     react: "npm:^18"
     react-dom: "npm:^18"
     react-icons: "npm:^5.0.1"
+    react-intersection-observer: "npm:^9.13.1"
```
Comment (jhw123) [inline yarn.lock:2952]: please update `yarn.lock` if you remove this package.

**Review by jhw123 — CHANGES_REQUESTED** [review]

File: public/members/default.png
Comment (alextio) [reply public/members/default.png:1]: You're right. I'll use this then. 
Frame 162

File: package.json (hunk 1 above)
Comment (alextio) [reply package.json:27]: react-markdown, yes. 
react-intersection-observer, no.

File: yarn.lock (hunk 2 above)
Comment (alextio) [reply yarn.lock:2952]: what would you say is the best way to update this yarn.lock file?
This article lists three ways: https://dev.to/ayc0/yarn-lock-how-to-update-it-1fa2

Comment (jhw123) [reply yarn.lock]: We use yarn4+. We can use dedupe

**Review by jhw123 — APPROVED** [review]

**Pull request merged**